Sources/SwiftTranslate/Models/OpenAIModel.swift

```diff
@@ -1,10 +0,0 @@
-//
-//  Copyright © 2024 Hidden Spectrum, LLC.
-//
-
-import Foundation
-
-
-public enum OpenAIModel: String {
```

Comment:
Might be missing it, where was this moved to?

Reply:
I removed it in favor of the OpenAI library's [Model](https://github.com/MacPaw/OpenAI/blob/fd13a41e987004d14f1793c570721953a2767b03/Sources/OpenAI/Public/Models/Models/Models.swift#L9) to allow more flexibility in specifying any model, e.g. if a new model is released it might work right away without needing to update swift-translate.